staging/src/k8s.io/apiserver/pkg/server/hooks.go

```diff
@@ -48,8 +49,11 @@ type PreShutdownHookFunc func() error
 type PostStartHookContext struct {
 	// LoopbackClientConfig is a config for a privileged loopback connection to the API server
 	LoopbackClientConfig *restclient.Config
-	// StopCh is the channel that will be closed when the server stops
+	// StopCh is the channel that will be closed when the server stops. Alternatively,
+	// Context can be used. StopCh is provided for present for code that hasn't been updated yet.
```

Comment:
deprecate?

Reply:
No strong opinion. We probably cannot ever be sure that it's not used and thus it has to stay forever, right?

I added a `Deprecated` comment explaining that. We had problems in the past with `Deprecated` comments triggering an avalanche of PRs replacing the deprecated API although there was no real benefit for doing so.